dockerhub_doc_config_update.sh

```diff
@@ -38,9 +38,6 @@ oses="alpine ubuntu centos windowsservercore-ltsc2022 nanoserver-ltsc2022 window
 # The image which is used by default when pulling shared tags on linux e.g 8-jdk
 default_linux_image="jammy"
 
-# shellcheck disable=SC2034 # used externally
```

Comment:
Are we sure this doesn't get used by an external script?

Reply:
I am not, is this file used outside the repo?

Reply:
https://github.com/search?q=org%3Aadoptium+hotspot_latest_tags&type=code tells me this can be deleted :-)